service/ArbitrageService.js

```diff
@@ -129,18 +159,22 @@ class ArbitrageService {
 
 			// swap usd from bsc to blxm
 			await this._bscContracts.poolContract.swapStablesToToken(usdSwapAmount);
-
+			logger.info("Swap " + ethers.utils.formatEther(usdcSwapAmount) + " USDC to BLXM in the Binance Smart Chain Network to perform arbitrage trade");
+			
 			arbitrageBlxmBalance = await this._bscContracts.blxmTokenContract.getTokenBalance(constants.ARBITRAGE_WALLET_ADDRESS);
 
+			logger.info("Add BLXM to the liquidity pool to complete arbitrage cycle");
 			return await this._bridgeAndSwapETH(amount, arbitrageBlxmBalance);
 		}
 	}
 
 	async _bridgeAndSwapETH(amount, abitrageBalance) {
 		// swap exact amount, if abitrage pool has enough tokens, swap minimal avaible otherwise
+		logger.info("Swapping " + ethers.utils.formatEther(amount) + " USDC to BLXM to perform arbitrage trade");
```

Comment:
Würde hier lieber den swap amount aus der nächsten Zeile loggen. Weil entweder der Wert vom AdjustmentValue geswapped und gebridged wird oder was noch in der Wallet verfügbar ist. 

Und das ist dann der swap amount